client/app/src/Controller/UserController.php

```diff
@@ -244,14 +256,14 @@ public function passwordForgottenAction(Request $request)
 
         return [
             'form' => $form->createView(),
+            'user' => $user->getId(),
```

Comment:
I don't think we ended up making use of this, so might be worth dropping.
I could be wrong though